src/test/migrations/steps/20241114204645-step-fix-datatable-grid-layouts-and-taxon-codes.ts

```diff
@@ -512,9 +512,59 @@ const _fixFraTaxonCodes = async (client: BaseProtocol) => {
   )
 }
 
+const _fixGrowingStockComposition2025HeaderRows = async (client: BaseProtocol) => {
+  const assessment = await AssessmentController.getOne({ assessmentName: AssessmentNames.fra }, client)
+
+  const schemaAssessment = Schemas.getName(assessment)
+
+  // Move 'header_1' row cols to 'header_0' row
+  await client.query(
+    `with target_row as (
+      select 
+          r.id as target_row_id
+      from ${schemaAssessment}.row r
+      join ${schemaAssessment}.table t on r.table_id = t.id
+      where r.props ->> 'index' = 'header_0'
+          and t.props ->> 'name' = 'growingStockComposition2025'
+    ),
+    cols_to_update as (
+      select 
+          c.id as col_id, 
+          target_row.target_row_id,
+          (c.props ->> 'index')::int + 3 as new_index -- number of cols in header_0 + index
+      from ${schemaAssessment}.col c
+      join ${schemaAssessment}.row r on c.row_id = r.id
+      join ${schemaAssessment}.table t on r.table_id = t.id
+      join target_row on true
+      where r.props ->> 'index' = 'header_1'
+          and t.props ->> 'name' = 'growingStockComposition2025'
+      )
+      update ${schemaAssessment}.col
+      set row_id = cols_to_update.target_row_id,
+        props = jsonb_set(
+            props,
+            '{index}',
+            to_jsonb(cols_to_update.new_index)
+        )
+      from cols_to_update
+      where ${schemaAssessment}.col.id = cols_to_update.col_id;
+
+      delete from ${schemaAssessment}.row
+      where id in (
+        select 
+            r.id
+        from ${schemaAssessment}.row r
+        join ${schemaAssessment}.table t on r.table_id = t.id
+        where r.props ->> 'index' = 'header_1'
+            and t.props ->> 'name' = 'growingStockComposition2025'
+      );`
```

Comment:
Any alternatives to handle moving cols from one row to another? 🤔

Reply:
Having two cols as in duplicating them? 

I was trying to simplify the query move the cols, but this was the most concise I could get it to.

Reply:
Ahh I see 😅 Allow me to explain:
This issue with the header:
![image](https://github.com/user-attachments/assets/f03f9266-bf57-4c1d-9654-b6958c67f8a2)

was there because the cols "FRA 2025 categories", "Scientific name", "Common name" are associated to a `row_id`, and "Million m³" with "% of total" are associated to a different `row_id`.
 When the actions are visible, we render an empty div for each header row, and because in the data we had two different rows for the same header, an extra empty div was added unnecessarily. 

So, in the query I change the `row_id` of "Million m³" and "% of total" to the `row_id` of  "FRA 2025 categories", "Scientific name" and "Common name". That way those are all in the same row. I then delete the row that is left with no cols. 

My question is if any other alternatives to that query come to mind